guides/tutorials/codelabs/kubernetes-v2-source-to-prod/index.md

```diff
@@ -0,0 +1,536 @@
+|-------|-------|
+| __Account__ | "staging-demo" |
+| __Cluster__ | "demo" |
+| __Manifest Source__ | "Artifact" |
+| __Expected Artifact__ | Must reference the `manifests/demo.yml` artifact. |
+| __Artifact Account__ | The GitHub artifact account configured above. |
+
+{% include figure
+   image_path="./configure-deploy-manifest-stage.png"
+%}
+
+Save the pipeline.
+
+# 3. Deploy Manifests to Staging
+
+Trigger the pipeline by pushing a commit to your repository. The pipeline
+should start in a few seconds -- when it completes, click __Details__ to see
+information about the execution:
+
+{% include figure
+   image_path="./staging-execution.png"
+   caption="There are a couple of things to notice here. First, in the top
+   right we get details about the commit that triggered this pipeline. Second,
+   in the __Deploy Status__ we can see what the YAML was that Spinnaker
+   deployed. Third, we see that the ConfigMap that we deployed was assigned
+   version `-v000`. This was done to ensure that you can statically reference
+   this ConfigMap, insulating any Pod that references it from accidental
+   changes."
+%}
+
+Next, let's see what this infrastructure looks like in Spinnaker. Navigate to
+the __Clusters__ tab, and select the blue Deployment object attached to the
+Replica Set shown below:
+
+{% include figure
+   image_path="./staging-v001.png"
+%}
+
+We can see in the __Artifact__ section on the right that we have bound our
+Docker image as well as our ConfigMap.
+
+Let's see what our application is serving. Run:
+
+```bash
+kubectl proxy --context $STAGING_CONTEXT
+```
+
+And then visit [the sample
+service](http://localhost:8001/api/v1/proxy/namespaces/default/services/spinnaker-demo:80/)
+in your browser. Let's make a change to this service, and configure Spinnaker
+to listen to Docker builds.
+
+# 4. Configure Docker Triggers
+
+__Important__: We need to configure DockerHub to build on __Tag__ events only,
+if we build on every commit, this particular setup will trigger both when
+manifests & code are changed at once. This can be configured under your
+Docker repository's __Build Settings__ tab as shown here:
+
+{% include figure
+   image_path="./docker-tag-only.png"
+   caption="This build rule will create a matching image tag each time you push
+   a git tag."
+%}
+
+Next, in Spinnaker, let's edit our Pipeline to allow Docker images to trigger a
+deployment:
+
+First, add a Docker expected artifact next to our Git expected artifact:
+
+{% include figure
+   image_path="./configure-docker-artifact.png"
+%}
+
+Next, add a _Webhook_ trigger to listen to build events from DockerHub. The
+_Docker_ trigger alone won't provide us with provenance information.
+
+{% include figure
+   image_path="./configure-docker-webhook.png"
+%}
+
+Finally, back in the "Deploy (Manifest)" stage configuration, list that the
+Docker artifact is expected to bind in this deployment:
+
+{% include figure
+   image_path="./bind-docker.png"
+%}
```

Comment:
In this image, the selected Docker artifact is in a fainter color. It looks l ike the github/file is the one selected.  Should we look at our highlighting in the UI? 
@sbwsg 

Might be a good thing in this case to circle the selection.

Reply:
Agree this is confusing.  Filed an issue: https://github.com/spinnaker/spinnaker/issues/2337